Description: Update default configuration
 Set agent logger default to system logging and stderr
 Enable 'include conf.d' directive
Author: Guillaume Bougard <gbougard@teclib.com>
Forwarded: not-needed
Last-Update: 2019-03-06

--- fusioninventory-agent-2.4.orig/etc/agent.cfg
+++ fusioninventory-agent-2.4/etc/agent.cfg
@@ -95,13 +95,13 @@ httpd-trust =
 #
 
 # Logger backend, either Stderr, File or Syslog (Stderr)
-logger = stderr
+logger = syslog,stderr
 # log file
 #logfile = /var/log/fusioninventory.log
 # maximum log file size, in MB
 #logfile-maxsize = 0
 # Syslog facility
-logfacility = LOG_USER
+logfacility = LOG_DAEMON
 # Use color in the console
 color = 0
 
@@ -131,5 +131,5 @@ debug = 0
 #   4. Package maintainers are encouraged to use this feature to avoid conflict
 #      during upgrades after configuration update
 #
-#include "conf.d/"
+include "conf.d/"
 #include "agent.local"
